Fix MavenReportException problem

我们鼓励使用英文，如果不能直接使用，可以使用翻译软件，您仍旧可以保留中文原文。
We recommend using English. If you are non-native English speaker, you can use the translation software. 


**Which Component**
eg. Nacos Config, RocketMQ

**Is your feature request related to a problem? Please describe.**
When I run command as follows `mvn clean install` under root directory in branch of 2.2.x, it will report warning information as below:
```
MavenReportException: Error while generating Javadoc: 
Exit code: 1 - /Users/raozihao/IdeaProjects/actual-project/spring-cloud-alibaba/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/parser/AbstractPropertySourceLoader.java:37: error: self-closing element not allowed
 * 1.inherit {@link AbstractPropertySourceLoader} ;<br/>
                                                   ^
/Users/raozihao/IdeaProjects/actual-project/spring-cloud-alibaba/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/parser/AbstractPropertySourceLoader.java:39: error: self-closing element not allowed
 * {@code org.springframework.boot.env.PropertySourceLoader=..}; <br/>
                                                                 ^
/Users/raozihao/IdeaProjects/actual-project/spring-cloud-alibaba/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/parser/NacosJsonPropertySourceLoader.java:76: warning: no @param for map
        protected Map<String, Object> reloadMap(Map<String, Object> map) {
                                      ^
/Users/raozihao/IdeaProjects/actual-project/spring-cloud-alibaba/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/parser/NacosJsonPropertySourceLoader.java:76: warning: no @return
        protected Map<String, Object> reloadMap(Map<String, Object> map) {
                                      ^
/Users/raozihao/IdeaProjects/actual-project/spring-cloud-alibaba/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/parser/NacosXmlPropertySourceLoader.java:43: error: unknown tag: conde
 * ({@conde DOCTYPE}) THE XML in a way that makes it difficult to customize the
    ^

Command line was: /Library/Java/JavaVirtualMachines/jdk1.8.0_291.jdk/Contents/Home/jre/../bin/javadoc @options @packages

Refer to the generated Javadoc files in '/Users/raozihao/IdeaProjects/actual-project/spring-cloud-alibaba/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/target/apidocs' dir.
```
```
MavenReportException: Error while generating Javadoc: 
Exit code: 1 - warning: unknown enum constant When.MAYBE
  reason: class file for javax.annotation.meta.When not found
/Users/raozihao/IdeaProjects/actual-project/spring-cloud-alibaba/spring-cloud-alibaba-starters/spring-cloud-starter-stream-rocketmq/src/main/java/com/alibaba/cloud/stream/binder/rocketmq/properties/RocketMQConsumerProperties.java:52: error: self-closing element not allowed
         * <p/>
           ^
/Users/raozihao/IdeaProjects/actual-project/spring-cloud-alibaba/spring-cloud-alibaba-starters/spring-cloud-starter-stream-rocketmq/src/main/java/com/alibaba/cloud/stream/binder/rocketmq/properties/RocketMQConsumerProperties.java:54: error: self-closing element not allowed
         * <p/>
           ^

Command line was: /Library/Java/JavaVirtualMachines/jdk1.8.0_291.jdk/Contents/Home/jre/../bin/javadoc @options @packages

Refer to the generated Javadoc files in '/Users/raozihao/IdeaProjects/actual-project/spring-cloud-alibaba/spring-cloud-alibaba-starters/spring-cloud-starter-stream-rocketmq/target/apidocs' dir.
```

**Describe the solution you'd like**
We need to fix the problem in branch of 2.2.x and check whether there are same problems in branch of 2021.x.

--- a/spring-cloud-alibaba-examples/nacos-example/nacos-config-example/src/main/java/com/alibaba/cloud/examples/example/ExtensionConfigExample.java
+++ b/spring-cloud-alibaba-examples/nacos-example/nacos-config-example/src/main/java/com/alibaba/cloud/examples/example/ExtensionConfigExample.java
@@ -28,7 +28,7 @@ import org.springframework.web.bind.annotation.RestController;
 /**
  * Example of extended configuration. When a configuration in the shared configuration
  * does not meet the requirements, the extended configuration can be used to override the
- * shared configuration. Priority: Main Configuration > Extended Configuration > Shared
+ * shared configuration. Priority: Main Configuration &gt; Extended Configuration &gt; Shared
  * Configuration.
  * @author lixiaoshuang
  */

--- a/spring-cloud-alibaba-examples/rocketmq-example/rocketmq-pollable-consume-example/src/main/java/com/alibaba/cloud/examples/pollable/RocketMQPollableConsumeApplication.java
+++ b/spring-cloud-alibaba-examples/rocketmq-example/rocketmq-pollable-consume-example/src/main/java/com/alibaba/cloud/examples/pollable/RocketMQPollableConsumeApplication.java
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.alibaba.cloud.examples.orderly;
+package com.alibaba.cloud.examples.pollable;
 
 import java.util.HashMap;
 import java.util.Map;

--- a/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/parser/AbstractPropertySourceLoader.java
+++ b/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/parser/AbstractPropertySourceLoader.java
@@ -35,9 +35,9 @@ import org.springframework.core.io.Resource;
  * Nacos-specific loader, If need to support other methods of parsing,you need to do the
  * following steps:
  * <p>
- * 1.inherit {@link AbstractPropertySourceLoader} ;<br/>
+ * 1.inherit {@link AbstractPropertySourceLoader};<br>
  * 2. define the file{@code spring.factories} and append
- * {@code org.springframework.boot.env.PropertySourceLoader=..}; <br/>
+ * {@code org.springframework.boot.env.PropertySourceLoader=..};<br>
  * 3.the last step validate.
  * </p>
  * Notice the use of {@link NacosByteArrayResource} .

--- a/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/parser/NacosXmlPropertySourceLoader.java
+++ b/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-config/src/main/java/com/alibaba/cloud/nacos/parser/NacosXmlPropertySourceLoader.java
@@ -40,7 +40,7 @@ import org.springframework.core.io.Resource;
 /**
  * Parsing for XML requires overwriting the default
  * {@link PropertiesPropertySourceLoader}, because it internally rigorously validates
- * ({@conde DOCTYPE}) THE XML in a way that makes it difficult to customize the
+ * ({@code DOCTYPE}) THE XML in a way that makes it difficult to customize the
  * configuration; at finally, make sure it's in the first place.
  *
  * @author zkz

--- a/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-sentinel/src/main/java/com/alibaba/cloud/sentinel/SentinelWebMvcConfigurer.java
+++ b/spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-sentinel/src/main/java/com/alibaba/cloud/sentinel/SentinelWebMvcConfigurer.java
@@ -27,7 +27,7 @@ import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
 /**
- * @author: chao.wu
+ * @author chao.wu
  */
 public class SentinelWebMvcConfigurer implements WebMvcConfigurer {
 

--- a/spring-cloud-alibaba-starters/spring-cloud-starter-stream-rocketmq/src/main/java/com/alibaba/cloud/stream/binder/rocketmq/properties/RocketMQConsumerProperties.java
+++ b/spring-cloud-alibaba-starters/spring-cloud-starter-stream-rocketmq/src/main/java/com/alibaba/cloud/stream/binder/rocketmq/properties/RocketMQConsumerProperties.java
@@ -49,9 +49,9 @@ public class RocketMQConsumerProperties extends RocketMQCommonProperties {
 
 	/**
 	 * The expressions include tags or SQL,as follow:
-	 * <p/>
+	 * <p>
 	 * tag: {@code tag1||tag2||tag3 }; sql: {@code 'color'='blue' AND 'price'>100 } .
-	 * <p/>
+	 * </p>
 	 * Determines whether there are specific characters "{@code ||}" in the expression to
 	 * determine how the message is filtered,tags or SQL.
 	 */

--- a/spring-cloud-alibaba-tests/spring-cloud-alibaba-test-support/src/main/java/com/alibaba/cloud/testsupport/Tester.java
+++ b/spring-cloud-alibaba-tests/spring-cloud-alibaba-test-support/src/main/java/com/alibaba/cloud/testsupport/Tester.java
@@ -19,7 +19,7 @@ package com.alibaba.cloud.testsupport;
 /**
  *
  * @author freeman
- * @date 2021.0.1.0
+ * @since 2021.0.1.0
  */
 public class Tester {
 

